### PR TITLE
IV fixes and some other stuff

### PR DESCRIPTION
**Pull request by mazinskihenry**
It'll fix the IVs and should help with some resuscitation issues

### CODE REVIEW

File: addons/pharma/stringtable.xml [hunk 1]
```diff
@@ -66,6 +53,9 @@
             <Czech>Změní který medický level je potřeba k zavedení IV/IO</Czech>
             <Japanese>IV/IOを設定するために必要な医療レベルを変更します</Japanese>
         </Key>
+        <Key ID="STR_ACEP_circulation_IV_DROP_ENABLE">
```
Comment (MiszczuZPolski) [inline addons/pharma/stringtable.xml]: Shouldn't it be kat instead of ACEP?

Comment (mazinskihenry) [reply addons/pharma/stringtable.xml]: AHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH

**Comment by YetheSamartaka** [comment]
LGTM.

File: addons/pharma/stringtable.xml [hunk 2]
```diff
@@ -66,6 +53,9 @@
             <Czech>Změní který medický level je potřeba k zavedení IV/IO</Czech>
             <Japanese>IV/IOを設定するために必要な医療レベルを変更します</Japanese>
         </Key>
+        <Key ID="STR_kat_pharma_IV_DROP_ENABLE">
```
Comment (Tomcat-SG) [inline addons/pharma/stringtable.xml:56]: more languages @MiszczuZPolski xD

**Review by Tomcat-SG — CHANGES_REQUESTED** [review]

File: addons/pharma/stringtable.xml [hunk 3]
```diff
@@ -66,6 +53,9 @@
...
+            <English>Enables IV/IO drop times</English>
```
Comment (Tomcat-SG) [inline addons/pharma/stringtable.xml:57]:
```suggestion
            <English>Enables IV/IO drop times</English>
            <German>Aktiviert das herausfallen von IOs/IVs</German>
            <Polish>Włącz wypadanie IV/IO</Polish>
            <Spanish>Habilitar los tiempos de caída de IV/IO</Spanish>
            <Chinese>启用IV/IO下降时间</Chinese>
            <Czech>Povolit čas vypadnutí pro IV/IO</Czech>
            <Korean>IV/IO 드롭 타임 활성화</Korean>
            <French>Activer les délais d'attente pour les IV/IO</French>
            <Turkish>IV/IO bırakma sürelerini etkinleştir</Turkish>
            <Italian>Abilitare i tempi di caduta IV/IO</Italian>
            <Japanese>IV/IOの滴下時間を有効にする</Japanese>
```

**Review by Tomcat-SG — CHANGES_REQUESTED** [review]

**Review by Tomcat-SG — APPROVED** [review]
LGTM

**Comment by YetheSamartaka** [comment]
LGTM.

**Pull request merged**